src/libraries/Microsoft.PowerFx.Connectors/Tabular/Services/TabularService.cs

```diff
@@ -17,10 +17,12 @@ public abstract class TabularService
 
         public bool IsInitialized => TableType != null;
 
+        public abstract bool IsDelegable { get; }
```

Comment:
>public abstract bool IsDelegable { get; } [](http://example.com/codeflow?start=8&length=41)

What does this mean?

Tabular Contract is delegable - just a question of which capabilities?

Reply:
If there are no capabilities this flag will be false